src/core/window/clipboard.js

```diff
@@ -2,11 +2,18 @@ import {removeElement} from '#core/dom';
 import {setStyles} from '#core/dom/style';
 
 /**
- * @param {Window} win
- * @param {string} text
- * @return {boolean}
+ * Copies provided text to clipboard using deprecated API: document.execCommand('copy')
+ * @param {Window} win Window context
+ * @param {string} text Text to copy
+ * @param {*} successCallback Executes when copying is successful
+ * @param {*} failCallback Executes when copying is failed
  */
-export function depricatedCopyTextToClipboard(win, text) {
+export function deprecatedCopyTextToClipboard(
```

Comment:
Given this is a non-async op, why the change to use callbacks? Also, if we stick with callbacks, we should definitely type them accordingly rather than using `*` any

Reply:
It looks to me like this is fine returning a boolean and the calling code can call callbacks as needed?

Reply:
> It looks to me like this is fine returning a boolean and the calling code can call callbacks as needed based on that return value?

I think you are correct. This approach seems much clear. I will revert it back to the original function returning `boolean` and the calling code can call `callbacks` as needed based on the return value.